packages/adapters-library/src/core/adapters/UniswapV2PoolForkAdapter.ts

```diff
@@ -634,13 +635,7 @@ export abstract class UniswapV2PoolForkAdapter implements IProtocolAdapter {
       factoryAddress,
       this.provider,
     )
-    const allPairsLength = 1 ?? Number(await factoryContract.allPairsLength())
-
-    if (allPairsLength > this.MAX_FACTORY_POOL_COUNT) {
```

Comment:
Next statement already picks a minimum between `MAX_FACTORY_POOL_COUNT` and `allPairsLength`, so this is not necessary.